entries/c/carleton.ca.json

```diff
@@ -0,0 +1,25 @@
+{
+  "Carleton University": {
+    "domain": "carleton.ca",
+    "img": "carleton.ca.png",
```

Comment:
Maybe this can be used?
```SVG
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 19.01 23.56"><path d="M9.5 0C6.43 0 3.6.45.48 1.58c-.35.1-.47.18-.47.55v9.4c.75 6.09 2.14 8.76 9.22 11.97.19.09.4.09.6 0 5.31-2.07 8.96-6.44 9.19-11.98V2.13c0-.25-.05-.4-.33-.49A27.98 27.98 0 0 0 9.51 0Z"/><path fill="#fffffe" d="M17.22 17.5a8.66 8.66 0 0 0-2.83-.88c-2.3-.35-4.83.1-6.78.42-3.47.45-5.63-.59-6.9-1.75-.11-.37-.22-.78-.3-1.15 1.3 1.34 3.52 2.5 7.27 1.2l-5.06-3.78 1.76-.48L2.74 7.2l3.6 1.05.32-2.28 3.33 3.6-1.47-6.42 2.62 1.4L12.74.2l.63.09 1.57 4.25 2.62-1.38-1.47 6.4 2.92-3.14c-.15 1.39.77 7.28-1.79 11.08zm-9.56.34c-3.57.29-5.66-.78-6.8-2.02.13.52.52 1.33.62 1.49 1.06.78 2.98 1.5 5.99 1.75 3.15.25 5.22 1.1 6.59 2.07.8-.62 1.52-1.32 2.16-2.1-1.54-.87-4.15-1.55-8.56-1.2"/><path fill="#d8232a" d="M19 11.5c.06-1.46-.01-2.9 0-4.37l-3.8 4.08 1.65-7.14-2.18 1.16L13.03.78 11.4 5.23 9.22 4.07l1.63 7.14-3.86-4.17-.25 1.82-3.17-.92 1.47 3.46-1.35.37 4.54 3.38c5.15-2 8.4-1.4 10.29-.63.3-.98.45-1.97.49-3.02z"/></svg>
```

Reply:
Thank you! I've changed the logo from png to the svg you provided and removed the img tag in the json entry in [commit #57ed5f9].(https://github.com/2factorauth/twofactorauth/pull/6860/commits/57ed5f9f43bc500dfe63eb6fac6895921042d490).